test/RunTests

```diff
@@ -1710,4 +1710,16 @@ printf '3 |f a b c |e x y z\n2 |f a y c |e x\n' | {VW} --oaa 3 -q ef --audit
 {VW} -k --oaa 10 --oaa_subsample 5 -c --passes 10 -d train-sets/multiclass --holdout_off
     train-sets/ref/oaa_subsample.stderr
 
-# Do not delete this line or the empty line above it
\ No newline at end of file
+# Test 191: malformed examples, onethread, strict_parse failure
+./negative-test.sh {VW} -d train-sets/malformed.dat --onethread --strict_parse
+    train-sets/ref/malformed-onethread-strict_parse.stderr
+
+# Test 192: malformed examples, strict_parse failure
+./negative-test.sh {VW} -d train-sets/malformed.dat --strict_parse
+    train-sets/ref/malformed-strict_parse.stderr
+
+# Test 193: malformed examples success
+{VW} -d train-sets/malformed.dat --onethread
```

Comment:
This test didn't pass for me without `--onethread`; my guess is there is some non-determinism in printing `parserWarning` messages from the parse thread and printing progress messages?

Reply:
Yes.  The parser thread runs asynchronous of the learning thread unless you use --onethread.